### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by nivabitbol**
This change suggested from customer following a support escelation.
Thanks,
Niv

### CODE REVIEW

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -405,6 +406,7 @@ Accept wildcard characters: False
 
 ### -DisableScanningNetworkFiles
 Indicates whether to scan for network files. If you specify a value of $False or do not specify a value, Windows Defender scans network files. If you specify a value of $True, Windows Defender does not scan network files. We do not recommend that you scan network files.
+Note: This parameter is specific for Quick and Scheduled scans.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:409]:
```suggestion
Note: This parameter applies specifically to quick and scheduled scans.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nivabitbol, kindly commit the suggested change. Many thanks.

**Comment by sdwheeler** [comment]
This change needs to be made for all versions, including 2022. Is the statement true for previous versions? If so, you need to update all versions.

**Comment by e0i** [comment]
@nivabitbol 

Please feel free to reopen the PR by updating all versions as suggested above.

**Pull request closed without merging**